app/Cargo.toml

```diff
@@ -8,5 +8,5 @@ edition = "2021"
 [dependencies]
 bindings = { path = "../bindings" }
 ethers = { git = "https://github.com/gakonst/ethers-rs", default-features = false }
```

Comment:
now that foundry uses crates relases, should we update here?

Reply:
done